src/util/irep_ids.txt

```diff
@@ -800,4 +800,5 @@ cprover_string_to_char_array_func
 cprover_string_to_lower_case_func
 cprover_string_to_upper_case_func
 cprover_string_trim_func
-cprover_string_value_of_func
\ No newline at end of file
+cprover_string_value_of_func
+handler
```

Comment:
where is the handler assigned? I only see a use of it here. In which PR is it introduced?